src/base/search/searchpluginmanager.cpp

```diff
@@ -172,7 +166,7 @@ void SearchPluginManager::updatePlugin(const QString &name)
 // Install or update plugin from file or url
 void SearchPluginManager::installPlugin(const QString &source)
 {
-    Logger::instance()->addMessage(tr("Installing search plugin from %1").arg(source), Log::INFO);
+    LogMsg(tr("Installing search plugin from %1").arg(source));
```

Comment:
The messages like "I'm going to do something...", "I started doing something...", "I'm doing something..." is useless in the Log in most cases. It is much more important to have here messages about the result of some actions, for example "Plugin *plugin_name* has been successfully updated."

Reply:
I'm adding back some messages and removing others. Let me know if you think there are some that should still be removed.